gradle.properties

```diff
@@ -0,0 +1,23 @@
+# Project-wide Gradle settings.
+# IDE (e.g. Android Studio) users:
+# Gradle settings configured through the IDE *will override*
+# any settings specified in this file.
+# For more details on how to configure your build environment visit
+# http://www.gradle.org/docs/current/userguide/build_environment.html
+# Specifies the JVM arguments used for the daemon process.
+# The setting is particularly useful for tweaking memory settings.
+org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
```

Comment:
Is the memory argument useful ? Otherwise, I'd keep only the encoding argument.

Also, I'd prefer it the comments in this file were removed.

Reply:
It is useful, but not mandatory. When building, Gradle will use its own JVM instance and this argument specifies how much memory it will get. Increasing the memory will improve the build performance, but I think my computer is not a good benchmark for that. As specified by https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory the default memory size is 512M. This project is rather small and efficient so I think it won't have a big impact (if at all) when this flag gets removed.
Additionally I will remove the Kotlin codestyle property since Kotlin isn't used.